### ISSUE TITLE
Use redbeat in celery 4.4.0rc2 failed

### ISSUE THREAD

**Issue by bigwolfdogroar** [sibson/redbeat#143]
I tested RedBeat in a project with celery 4.3 and 4.4. 
Something wrong happened.
RedBeat load tasks success both in different env.
But only send task to in celery 4.3.

Additionally, beat in Celery 4.4 have a good feature named BeatLazyFunc didn't work in RedBeat.I just fixed this feature in my project.

So, can you update this project to suits celery 4.4?

python: 3.7.3
RedBeat: v0.13.0
Celery: 4.3 and 4.4.0rc2

**Comment by noamkush** [comment]
I just tested this and redbeat worked fine, sending tasks as expected. Do you have a way to reproduce the issue?

**Comment by Lorry1123** [comment]
Same problem too.
Redbeat seems failed to send task to broker with celery 4.4.0. According to the logs, it just registered all tasks and then wake up every 5 minutes.
```
[2020-01-09 16:01:16,822][DEBUG][/root/.pyenv/versions/3.7.4/envs/sms3/lib/python3.7/site-packages/celery/beat.py:633][start][Standalone]: beat: Waking up in 5.00 minutes.
```

When using celery 4.3.0, it works fine.

python: 3.7.4
redbeat: 0.13.0
celery: 4.4.0 and 4.3.0

**Comment by yixikkjkj** [comment]
Celery fix a bug in 4.4.0rc2, but it becomes a bug in redbeat:
Fix bug in remaining time calculation in case of DST time change [#5411](https://github.com/celery/celery/pull/5462)

In redbeat, datetime is stored in UTC, but now Celery4.4.0 checks timezone now.
I hope this is helpful.

**Comment by sempr** [comment]
I created a pull request,  #150 will solve this issue forever.

**Comment by chen-zhuohan** [comment]
thanks！

**Comment by jlaine** [comment]
I think this issue can be closed, #165 (which replaces #150) was merged
